Library/Formula/vc4asm.rb

```diff
@@ -0,0 +1,16 @@
+class Vc4asm < Formula
+  homepage "http://maazl.de/project/vc4asm/doc/index.html"
+  url "https://github.com/maazl/vc4asm/archive/V0.1.7.tar.gz"
+  sha1 "20594439e9011dbd2175903754735cafbd01f029"
+
+  def install
+    rm_rf "bin"
+    mkdir "bin"
+    mkdir "obj"
+    cd "src" do
+       system "make"
+    end
+    bin.install Dir["bin/*"]
+    share.install Dir["share/*"]
+  end
+end
```

Comment:
It doesn't have a nice testing suite, it has some assembly files that are supposed to fail but not sure if it makes sense here.